components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -12,7 +12,7 @@ import ProportionalAreaChart from '../../ProportionalAreaChart';
 import { P, Span } from '../../Text';
 const Chart = dynamic(() => import('react-apexcharts'), { ssr: false });
 
-const getChartOptions = (intl, startDate, endDate, hostCreatedAt) => {
+const getChartOptions = category => {
```

Comment:
:ice_cream: I would find it clearer to name it like in the other places

```suggestion
const getChartOptions = timeUnit => {
```